cypress.config.ts

```diff
@@ -0,0 +1,15 @@
+import { defineConfig } from "cypress";
+
+const library = process.cwd().split("/").slice(-1)[0].replace("-stub", "");
+console.log(`Running Cypress integration tests for library: ${library}`);
+
+module.exports = defineConfig({
+  integrationFolder: "../../cypress/integration",
+  fixturesFolder: false,
+  pluginsFile: false,
+  supportFile: false,
+  video: false,
+  env: {
+    UI_LIBRARY: library,
```

Comment:
Added this to use `Cypress.env('UI_LIBRARY')` to conditionally skip certain test suites